test/integration/migrate/index.js

```diff
@@ -331,6 +331,55 @@ module.exports = function(knex) {
       });
     });
 
+    describe('knex.migrate.rollback - all', () => {
+      before(() => {
+        return knex.migrate
+          .latest({
+            directory: ['test/integration/migrate/test'],
+          })
+          .then(function() {
+            return knex.migrate.latest({
+              directory: [
+                'test/integration/migrate/test',
+                'test/integration/migrate/test2',
+              ],
+            });
+          });
+      });
+
+      it('should delete all batches from the migration log', function() {
```

Comment:
why not use arrow function?

Reply:
Because of [this](https://mochajs.org/#arrow-functions)?

Reply:
well yeah when it's needed, but when it's not?..